lib/build-all-ng.sh

```diff
@@ -149,6 +149,8 @@ build_main ()
 	[[ $BUILD_MINIMAL == yes ]] && upload_image=${upload_image}_minimal
 
 	touch "/run/armbian/${VENDOR}_${BOARD^}_${BRANCH}_${RELEASE}_${DESKTOP_ENVIRONMENT}_${BUILD_DESKTOP}_${BUILD_MINIMAL}.pid";
+    LOG_SUBPATH="${VENDOR}_${BOARD^}_${BRANCH}_${RELEASE}_${DESKTOP_ENVIRONMENT}_${BUILD_DESKTOP}_${BUILD_MINIMAL}"
```

Comment:
LOG_SUBPATH="debug/${VENDOR}_${BO..."

Also when DESKTOP_ENVIRONMENT is not defined we have double _

drwxrwsr-x 2 root  sudo   4096 Jul 27 16:54 Armbian_Bananapi_current_buster__no_no
drwxrwsr-x 2 root  sudo   4096 Jul 27 16:54 Armbian_Bananapi_current_buster_xfce_yes_no

Content is present.

Reply:
yeah I was aware of the double __, figured since the code wasn't accommodating that for pid files etc, to not bother right now.